src/functions/utils.php

```diff
@@ -638,4 +638,58 @@ function tribe_register_rest_route( $namespace, $route, $args = array(), $overri
 		$args = apply_filters( 'tribe_register_rest_route_args', $args, $namespace, $route, $override );
 		return register_rest_route( $namespace, $route, $args, $override );
 	}
-}
\ No newline at end of file
+}
+
+if ( ! function_exists( 'tribe_installed_before' ) ) {
```

Comment:
I don't think we need these anymore. These are not supposed to replaced right?

Reply:
No, I assumed they were in case of both versions of common being run - but that has been handled so 🔪